src/services/Index.php

```diff
@@ -13,6 +13,7 @@
 
 class Index extends Component
 {
+    private static $STORAGE_FOLDER = CRAFT_BASE_PATH . '/web';
```

Comment:
I wonder if we could consolidate the `$STORAGE_FOLDER` and `$TARGET_FOLDER` constants to something like `$OUTPUT_PATH` so it's a little more abstract